lib/rules/require-padding-newlines-before-line-comments.js

```diff
@@ -83,13 +83,9 @@ module.exports.prototype = {
         var allowFirstAfterCurly = this._allowFirstAfterCurly;
 
         file.iterateTokensByType('Line', function(comment) {
-            if (comment.loc.start.line === 1) {
```

Comment:
I'd say
```
#!/usr/bin/env

// line comment
// etc.
```
But
```
// line comment
// etc.
```

Well, it's very rare and hard to implement case (unless we have a isShebanged flag in jsfile). Nvm! ;-)